src/status_im/activity_center/core.cljs

```diff
@@ -6,6 +6,14 @@
             [status-im.utils.fx :as fx]
             [taoensso.timbre :as log]))
 
+;;;; Misc
+
+(fx/defn process-notification-failure
+  {:events [:activity-center/process-notification-failure]}
+  [_ notification-id action error]
+  (log/warn (str "Failed to " action)
+            {:notification-id notification-id :error error}))
```

Comment:
Can this be a `js/alert` to more visible to users/designers ?

Reply:
I have the same question @ibrkhalil. These failures should almost never happen, because the failure would mean an error from the RPC call because of an invalid RPC payload or because of a bug in status-go, both quite rare. The log is more for the end-user to be able to report a bug with more useful logs, but it's certainly miles away from a good UX.

Reply:
I'm going to bring this discussion to designers in the near future. Thanks for raising this concern.